src/components/shared/MemberReputation/MemberReputation.tsx

```diff
@@ -41,24 +42,35 @@ const MemberReputation = ({
   onReputationLoaded = () => null,
   showIconTitle = true,
 }: Props) => {
-  const { data: userReputationData } = useUserReputationQuery({
-    variables: { address: walletAddress, colonyAddress, domainId, rootHash },
+  const { data: userReputationData } = useQuery(gql(getUserReputation), {
+    variables: {
+      input: {
+        colonyAddress,
+        walletAddress,
+        domainId,
+        rootHash,
+      },
+    },
```

Comment:
This is is a `shared` _(old Dapp `core`)_ type component.

They should be re-usable, and mostly for display purposes. They should not handle fetching data internally. All that data fetching should happen in the `common` type components of the CDapp.

Now I know this was this way when you brought it over from the Dapp, but this would be a good opportunity to fix that.

Please refactor this, so that the data is fetched outside from it _(most likely in `UserNavigation`)_, and this component just ingests it and displays it (maybe handle formatting